crates/aptos-dkg/src/weighted_vuf/bls/mod.rs

```diff
@@ -82,13 +82,14 @@ impl WeightedVUF for BlsWUF {
         proof: &Self::ProofShare,
     ) -> anyhow::Result<()> {
         let hash = Self::hash_to_curve(msg);
-        // TODO: Use Fiat-Shamir
-        let coeffs = random_scalars(apk.len(), &mut thread_rng());
+        // TODO: Use Fiat-Shamir instead of random_scalar
+        let coeffs = get_powers_of_tau(&random_scalar(&mut thread_rng()), apk.len());
```

Comment:
How does this affect the performance in production?

Reply:
It does not. This is BLS.